sdcm/data_operations/data_operation.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+import logging
+import random
+import threading
+import time
+from functools import cached_property
+
+from sdcm.utils.user_profile import get_profile_content, get_table_by_stress_cmd, get_keyspace_by_stress_cmd
+
+
+class ResurrectionDataFound(Exception):
+    pass
+
+
+class SchemaManage:
+    """
+    Manage the test schema.
+    Run cassandra-stress.
+    Keep keyspace, table, view names
+    """
+    DEFAULT_STRESS_WRITE_COMMAND = ("cassandra-stress user profile=/tmp/data_operation_default.yaml n=100000 ops'(insert=10)' cl=QUORUM "
+                                    "-mode native cql3 -rate threads=30")
+
+    def __init__(self, tester, stress_cmd: str = DEFAULT_STRESS_WRITE_COMMAND):
+        self.main_stress_cmd = stress_cmd
+        _, self.main_profile_content = get_profile_content(stress_cmd=stress_cmd)
+        self.tester = tester
+        self.cluster = tester.db_cluster
+
+    @cached_property
+    def keyspace_name(self):
+        return get_keyspace_by_stress_cmd(stress_cmd=self.main_stress_cmd, profile_content=self.main_profile_content)
+
+    @cached_property
+    def table_name(self):
+        return get_table_by_stress_cmd(stress_cmd=self.main_stress_cmd, profile_content=self.main_profile_content)
+
+    @cached_property
+    def view_names(self):
+        with self.cluster.cql_connection_patient(self.cluster.nodes[0]) as session:
+            result = list(session.execute(f"select view_name from system_schema.views where keyspace_name='{self.keyspace_name}' and "
+                                          f"base_table_name = '{self.table_name}' ALLOW FILTERING"))
+
+            return [row.view_name for row in result] if result else None
+
+    @cached_property
+    def partition_keys(self):
+        with self.cluster.cql_connection_patient(self.cluster.nodes[0]) as session:
+            result = list(session.execute(f"select column_name from system_schema.columns where keyspace_name='{self.keyspace_name}' "
+                                          f"and table_name='{self.table_name}' and kind = 'partition_key' ALLOW FILTERING",
+                                          timeout=120))
+
+            return [row.column_name for row in result] if result else None
+
+    def create_default_keyspace_query(self, replication_factor, replication_strategy="SimpleStrategy"):
+        self.tester.create_keyspace(keyspace_name=self.keyspace_name,
+                                    replication_factor=replication_factor,
+                                    replication_strategy=replication_strategy)
+
+    def run_user_profile(self, stress_cmd=None):
+        self.tester.run_stress(stress_cmd=stress_cmd or self.DEFAULT_STRESS_WRITE_COMMAND)
+
+
+class DataOperations:
```

Comment:
DataOperation is a bit vague since all the function here are for one purpose on one type of data validation, why not call it like that ? 

once we have more of those we can see how to share some of the functionality, with functions, with a base class, with groups of specific classes, for now it's seem a be detached

Reply:
we have data_validation already.
The meaning here is change data, manipulate with data